frontend/projects/ui/src/app/app/menu/menu.component.ts

```diff
@@ -54,22 +53,25 @@ export class MenuComponent {
   readonly showEOSUpdate$ = this.eosService.showUpdate$
 
   readonly updateCount$: Observable<number> = combineLatest([
-    this.marketplaceService.getMarketplace$(),
-    this.patch.watch$('package-data'),
+    this.marketplaceService.getMarketplace$(true),
+    this.patch.watch$('package-data').pipe(first()),
```

Comment:
Curious why this wasn't `first()` before. Without `first()` I would expect this count to go up forever, since it would fire every time package-data is updated. @waterplea do you know?

Reply:
The count will not go up, it will just get refreshed when an update appears since it recalculates the number on each emit. This way updates count should actually go down when we install one of the updates available. At least that's what I think it was supposed to do.